### DISCUSSION TITLE
Need help migrating from v1 to v2 wallet connect? Wallet will not connect!

### DISCUSSION THREAD

**Discussion by Scottcarp1** [WalletConnect/walletconnect-monorepo#3733] [Developer Support] [answered]
Need help migrating from v1 to v2?
Step by step instructions would be wonderful if possible? I use a iPhone and have a desktop. Any help is much appreciated! I have v1 using trust wallet and need to migrate to v2 on via trust wallet?

**Comment by glitch-txs** [comment, marked as answer]
Unless you're building a Dapp, you don't need to migrate since Trust Wallet already supports WalletConnect v2.

**Reply by Scottcarp1** [reply]
Hi Tks for the reply. So Im having trouble connecting my trust wallet to shibaswap via wallet connect and every time I get a failed to connect. I thought I needed to upgrade there wallet. Not having any luck. Any ideas what to do next?

**Reply by glitch-txs** [reply]
That website is using the v1 protocol, they need to update to v2 on their end

**Reply by Scottcarp1** [reply]
Appreciate that info! I will check with them and get back to you about it.

**Reply by Scottcarp1** [reply]
So I am still not able to connect to their platform. Still says failed to connect. Any ideas I can’t find a spot to email them or and FAQ. It’s been like this for awhile. Do you know, is this normal thing. I can’t be the only person not able to connect and see there funds right? Tks for all for time helping me. Appreciate you!

**Reply by glitch-txs** [reply]
We'll try to contact them

**Reply by glitch-txs** [reply]
I got this response from discord

> The dev team is aware, and this has been postponed to v2, since ShibaSwap 2.0 will be released in the near future. Still no release date announced

**Comment by Scottcarp1** [comment]
Thank you so much for that information! I will be in contact in the future. I appreciate all your help!

**Comment by Scottcarp1** [comment]
﻿
Thank you so much for that information! I will be in contact in the future. I appreciate all your help!
